### ISSUE TITLE
Pyccel crashes when given empty class

### ISSUE THREAD

**Issue by jalalium** [pyccel/pyccel#1302]
### Describe the bug
Pyccel crashes in the syntactic stage when given an empty class. This is due to the assumption that a class will always have members.
### To Reproduce
Provide code to reproduce the behavior:
```python
#$ header class A(public)

if __name__ == "__main__":
    class A(object):
        pass
```

### Error details
Provide the generated code, or the error message:
```bash/fortran/c
  File "/../pyccel/parser/syntactic.py", line 832, in _visit_ClassDef
    attributes = [a.var for a in methods[0].arguments]
IndexError: list index out of range
```

### Expected behavior
The expected behavior is to output an adequate error message.

### Language
Both C and Fortran, the error happens earlier

### Additional context
This bug is easy to fix. Although classes are not a priority at the moment, `Pyccel` should never crash. Is it worth fixing it now?

**Comment by EmilyBourne** [comment]
I disagree that classes is not a priority. I see no reason to leave it to later as long as you know what is expected
